### DISCUSSION TITLE
How should a shebang for rustpython be like?

### DISCUSSION THREAD

**Discussion by glyh** [RustPython/RustPython#3587] [Q&A]
As title, I'm trying to use rustpython as scripting environment, here's what I tired:
```
#!/usr/bin/env -S wapm run rustpython
```

But this is not working, here's the error message: 
```
[2022-04-01T07:28:43Z ERROR rustpython] can't open file './[filename]]': No such file or directory
```

Could someone tell me how to fix this? Thanks a lot!

**Comment by coolreader18** [comment]
You might need to pass `--dir` to wapm-run
